main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -2740,6 +2740,11 @@ object Resolver {
     * Looks up the qualified name in the given root.
     */
   private def tryLookupQualifiedName(qname0: Name.QName, env: ListMap[String, Resolution], ns0: Name.NName, root: NamedAst.Root): Option[List[NamedAst.Declaration]] = {
+    // Fail if the name is un-qualified.
+    if (qname0.isUnqualified) {
```

Comment:
This seems like a logical error rather than a runtime error.

Reply:
Looking at the issue: `use x` should have a specific error like "can only use qualified names". But returning None here gives a NotFound error